api/user/author/authors.py

```diff
@@ -0,0 +1,48 @@
+from flask import Blueprint, jsonify, request
+
+from api import db
+from api.user.author.model import Author
+from api.utils import get_pagination_params
+
+# note: this blueprint is usually mounted under /authors URL prefix
+authors_bp = Blueprint("authors", __name__)
+
+
+# todo: how to interpret it?
+# URL: ://service/authors/
+#
+#     GET [local, remote]: retrieve all profiles on the server (paginated)
+#         page: how many pages
+#         size: how big is a page
+#
+# Example query: GET ://service/authors?page=10&size=5
+#
+#     Gets the 5 authors, authors 45 to 49.
+
+
+@authors_bp.route("/", methods=["GET"])
+def get_authors():
+    args = request.args
+    paginator = get_pagination_params()
+    authors = Author.query.paginate(page=1, per_page=paginator.size).items
+    return jsonify(authors)
+
+
+@authors_bp.route("/admin", methods=["POST"])
```

Comment:
This is for our own need to test easily in postman. I plan to put this behind an authentication after we have one.